apps/webhook/Dockerfile

```diff
@@ -1,36 +1,67 @@
-FROM node:16-buster-slim
+FROM nikolaik/python-nodejs:python3.10-nodejs16-slim as dev_base
```

Comment:
What's the reason behind choosing this docker image? Do we need Python inside the image?

Reply:
It was needed to compile by one of our dev dependencies. I think new relic or one of those. Would definitely prefer the simple image here.